plugins/core/IO/qLASIO/src/LasVlr.cpp

```diff
@@ -25,13 +26,13 @@
 #include <ccPointCloud.h>
 // System
 #include <algorithm>
-#include <cstring>
 
 LasVlr::LasVlr(const laszip_header& header)
 {
 	const auto vlrShouldBeCopied = [](const laszip_vlr_struct& vlr)
 	{
-		return !LasDetails::IsLaszipVlr(vlr) && !LasDetails::IsExtraBytesVlr(vlr);
+		// Currently, we avoid copying back the COPC VLR (due to potential modifications to the file)
```

Comment:
Not sure to understand the comment (or at least, why it's here)

Reply:
We exclude COPC's VRL to prevent inconsistencies when rewriting the file, as the user might have altered the file within CC.

Reply:
And it's hard to create a new one I guess?

Reply:
if we modify the number of points within CC (and the BBox in some extent), we have to (re)create a Nested Octree like data structure. it's basically what PotreeConverter and Entwine/Untwine generate. It should not be too hard to implement a "basic" In core indexer in CC but it is out of my scope in the short term.